src/app/wizard/step/provider-settings/provider/extended/kubevirt/component.ts

```diff
@@ -125,15 +125,17 @@ export class KubeVirtProviderExtendedComponent extends BaseFormValidator impleme
   }
 
   addPreAllocatedDataVolume(): void {
+    const pattern = Validators.pattern(/^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/);
```

Comment:
provide some valid testing values? can we move this regex too this path?

**Path:** src/app/shared/validators/others.ts 

can be utilized at multiple places? thoughts